### PR TITLE
GCP: Creating external IP Addresses

### PR DESCRIPTION
**Pull request by rambleraptor**
##### SUMMARY
Adding support for GCP Compute IP Addresses

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
gcp_compute_address

##### ANSIBLE VERSION
```
2.6
```

##### ADDITIONAL INFORMATION
This was autogenerated using [Magic Modules](https://github.com/GoogleCloudPlatform/magic-modules)

### CODE REVIEW

**Comment by rambleraptor** [comment]
@ryansb please take a look

**Comment by GwenaelPellenArkeup** [comment]
Hello. 

why create a new module?

gce_eip.py can create an External IP.

Best regards.

**Comment by ryansb** [comment]
the new `gcp_*` series of modules are autogenerated from the new version of the GCP API, so it doesn't have the same SDK requirements and sometimes differs from the parameters accepted in the existing modules (for services that have existing modules). 

The idea is to use the [Magic Modules](https://github.com/GoogleCloudPlatform/magic-modules) code generator to more completely cover the GCP services. Since Magic Modules is also open source, we can review the modules it generates and make sure they meet Ansible standards for docs, usability, and such.

File: lib/ansible/modules/cloud/google/gcp_compute_address.py [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+---
+module: gcp_compute_address
+description:
+    - Represents an Address resource.
+      Each virtual machine instance has an ephemeral internal IP address and,
+      optionally, an external IP address. To communicate between instances on
+      the same network, you can use an instance's internal IP address. To
+      communicate with the Internet and instances outside of the same network,
+      you must specify the instance's external IP address.
+      Internal IP addresses are ephemeral and only belong to an instance for
```
Comment (ryansb) [inline lib/ansible/modules/cloud/google/gcp_compute_address.py]: For a paragraph break (which seems like what this line is supposed to be) make the string a new list item.

File: lib/ansible/modules/cloud/google/gcp_compute_address.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+            - The static external IP address represented by this
+              resource. Only IPv4 is supported.
+        required: false
+    description:
+        description:
+            - An optional description of this resource.
+        required: false
+    name:
+        description:
+            - Name of the resource.
```
Comment (ryansb) [inline lib/ansible/modules/cloud/google/gcp_compute_address.py]: It'd be nice to include any restrictions on the name (can it have spaces? Dashes? Umlaüts?)

**Review by ryansb — APPROVED** [review]

**Review by s-hertel — APPROVED** [review]

**Pull request merged**